UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Common.UnitTests/Helpers/FileHelperTest.cs

```diff
@@ -28,7 +28,25 @@ public void WhenFormatSizeIsCalled_ThenShouldReturnSizeInKB()
         public void WhenFormatSizeIsCalled_ThenShouldReturnSizeInMB()
         {
             long bytes = 1234567;
-            string expected = "1.2 MB";
+            string expected = "1 MB";
+            string result = FileHelper.FormatSize(bytes);
+            Assert.AreEqual(expected, result);
+        }
+
+        [Test]
+        public void WhenFormatSizeIsCalled_ThenShouldRoundUpValue()
+        {
+            long bytes = 1834567;
+            string expected = "2 MB";
```

Comment:
These (and others) can be consts please:
```suggestion
            const long bytes = 1834567;
            const string expected = "2 MB";
```

Reply:
made variables constant